src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
+
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+        System.out.print("Got it. I've added this task:\n" + this + '\n');
+    }
+
+    public String toString() {
```

Comment:
Perhaps consider including @Override when overriding the toString method. This could apply for the other classes as well.

Reply:
Thank you! Would add it to my code.